src/services/TechRecordsService.ts

```diff
@@ -215,6 +215,11 @@ class TechRecordsService {
   }
 
   private createAndArchiveTechRecord(techRecord: { vin: string, techRecord: ITechRecord[] }, msUserDetails: any) {
+    const {statusCode} = techRecord.techRecord[0];
+    delete techRecord.techRecord[0].statusCode;
```

Comment:
Are you sure you want to delete the statusCode from the tech-record here?

Reply:
I'm using the statusCode to uniquely identify the tech-record to update. I'm deleting the statusCode from the payload not from the actual record that will be saved in DB, because currently we're not allowed to update the statusCode on a tech-record and statusCode is not part of the validation schema and it will throw an error if statusCode is present on the payload.